bc_obps/reporting/migrations/0086_update_storage_of_petroleum_products_shcema.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 5.2 on 2025-04-16 18:32
```

Comment:
you'll need to run `poetry install` to make sure you have the right django version installed